tests/karma.conf.local.js

```diff
@@ -4,7 +4,16 @@ module.exports = function (config) {
     // https://karma-runner.github.io/5.2/config/browsers.html
     browsers: [
       'FirefoxHeadless',
-      'ChromeHeadless'
+      // In general, Linux distros provides and updates Chromium only,
+      // whereas Windows and macOS users tend to have auto-updating Google Chrome.
+      // For developer convenience, treat these as the same as pick the are most likely
```

Comment:
Some typos have garbled the syntax here: "treat these as the same as pick the are" ??